code/items/equipments/entities/C4Entity.cs

```diff
@@ -0,0 +1,150 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+using Sandbox;
+using Sandbox.UI;
+
+using TTTReborn.Player;
+
+namespace TTTReborn.Items
+{
+    [Library("ttt_c4_ent"), Hammer.Skip]
+    public partial class C4Entity : Prop, IUse
+    {
+        private string ModelPath => "models/entities/c4.vmdl";
+
+        [Net]
+        public int AttachedBone { get; set; } = -1;
```

Comment:
Might be a good place for a comment, why is `attachedbone` set to `-1`?

Reply:
Added a comment with some detail.

Considering dropping the [Net] as I believe the assignment of this is only used on the server.